helpers/csvref2json.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/python3
```

Comment:
It seems like a copy of csvref2json. Please integrate your changes in the current script.